### PR TITLE
Fix error code coming back for generic geocode issues

### PR DESCRIPTION
**Pull request by bmallred**
Resolves truetandem/e-QIP-prototype#2693

Fixes a couple of issues:
 - does not validate on non-existing states
 - display state code by default (as was initial desire)
 - fix generic error code from API

## Generic message
image

## Handling of non-existing state
image

### CODE REVIEW

**Comment by floresj** [comment]
selection_820

**Comment by floresj** [comment]
selection_821

File: src/validators/alcoholorderedcounseling.test.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe('ordered counseling component validation', function () {
                     country: { value: 'United States' },
                     street: '1234 Some Rd',
                     city: 'Arlington',
-                    state: 'Virginia',
```
Comment (floresj) [inline src/validators/alcoholorderedcounseling.test.js]: Didn't realize we could also do full state names before :taco:

Comment (bmallred) [reply src/validators/alcoholorderedcounseling.test.js]: We've gone back-and-forth a few times on what is displayed/accepted on dropdown lists but definitely thought this was not the direction we went :rabbit:

**Pull request merged**